lib/ScriptManager.js

```diff
@@ -4,7 +4,8 @@ var events = require('events'),
     convert = require('./convert.js');
 
 // see Blink inspector > ContentSearchUtils.cpp > findMagicComment()
-var SOURCE_MAP_URL_REGEX = /\/\/[@#][ \t]sourceMappingURL=[ \t]*([^\s'"]*)[ \t]*$/m;
+var SOURCE_MAP_URL_REGEX = /\/\/[@#][ \t]sourceMappingURL=[ \t]*([^\s'"]*)[ \t]*$/m,
+    INLINE_SOURCE_MAP_REGEX = /^(data:application\/json;base64,)(.*)$/;
```

Comment:
I think your regexp is too strict, the URI can have other header and still stay valid:

```
data:application/json;charset=utf-8;base64,...data...
data:application/json,...url-encoded-data...
```

In the past, I wrote a data-URI parser specifically for Node Inspector, see https://www.npmjs.com/package/strong-data-uri. Please consider using that module instead of re-implementing the parser again.